test/wsteth-price-feed.ts

```diff
@@ -0,0 +1,107 @@
+import { ethers, exp, expect } from './helpers';
+import {
+  SimplePriceFeed__factory,
+  SimpleWstETH__factory,
+  WstETHPriceFeed__factory
+} from '../build/types';
+
+export async function makeWstETH({ stEthPrice, tokensPerStEth }) {
+  const SimplePriceFeedFactory = (await ethers.getContractFactory('SimplePriceFeed')) as SimplePriceFeed__factory;
+  const stETHPriceFeed = await SimplePriceFeedFactory.deploy(stEthPrice, 8);
+
+  const SimpleWstETHFactory = (await ethers.getContractFactory('SimpleWstETH')) as SimpleWstETH__factory;
+  const simpleWstETH = await SimpleWstETHFactory.deploy(tokensPerStEth);
+
+  const wstETHPriceFeedFactory = (await ethers.getContractFactory('WstETHPriceFeed')) as WstETHPriceFeed__factory;
+  const wstETHPriceFeed = await wstETHPriceFeedFactory.deploy(
+    stETHPriceFeed.address,
+    simpleWstETH.address
+  );
+  await wstETHPriceFeed.deployed();
+
+  return {
+    simpleWstETH,
+    stETHPriceFeed,
+    wstETHPriceFeed
+  };
+}
+
+const testCases = [
```

Comment:
Weird edge case to maybe test is if the price returned is negative (Chainlink returns `int256` for price). I don't think it really matters since Comet will just revert if it's a negative price, but let's make sure we do return a negative price back.

Reply:
good idea. added a negative price to the test cases